app/src/main/java/asd/org/ahorcado/models/AbstractMatch.java

```diff
@@ -53,4 +53,8 @@ public boolean afterExecute(char letter) throws MatchLostException {
     public String getNewWord() {
         return guessWord.getNewWord();
     }
+
+    public void initialGame(){
+        setLife(6);
```

Comment:
Yes! I think the approach about create properties in xml file is the best option. We can create a new issues!